Tests/RxSwift/MapManyTests.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  MapManyTests.swift
+//  RxSwiftExt
+//
+//  Created by Joan Disho on 06.05.18.
+//  Copyright © 2018 RxSwiftCommunity. All rights reserved.
+//
+
+import XCTest
+import RxSwift
+import RxTest
+
+fileprivate struct SomeModel: CustomStringConvertible {
+    let number: Int
+    var description: String { return "#\(number)" }
+
+    init(_ number: Int) {
+        self.number = number
+    }
+}
+
+class MapManyTests: XCTestCase {
+    fileprivate func runAndObserve<C: Collection>(_ sequence: Observable<C>) -> TestableObserver<C> {
```

Comment:
can you get rid of `fileprivate` here (and also on top) ? 
It's irrelevant given the fact its part of a test suite, and the method is part of the class itself.